product_docs/docs/pgd/5/postgres-configuration.mdx

```diff
@@ -78,4 +76,4 @@ timestamp to use.
 
 ### `max_prepared_transactions`
 
-Needs to be set high enough to cope with the maximum number of concurrent prepared transactions across the cluster due to explicit two-phase commits, CAMO, or Eager transactions. Exceeding the limit prevents a node from running a local two-phase commit or CAMO transaction and prevents all Eager transactions on the cluster. You can set this only at Postgres server start.
+Needs to be set high enough to cope with the maximum number of concurrent prepared transactions across the cluster due to explicit two-phase commits, CAMO, or Eager transactions. Exceeding the limit prevents a node from running a local two-phase commit or CAMO transaction and prevents all Eager transactions on the cluster. You can set this parameter only at Postgres server start.
```

Comment:
```suggestion
Needs to be set high enough to cope with the maximum number of concurrent prepared transactions across the cluster due to explicit two-phase commits, CAMO, or Eager transactions. Exceeding the limit prevents a node from running a local two-phase commit or CAMO transaction and prevents all Eager transactions on the cluster. You can only set this parameter at Postgres server start.
```

Reply:
Hi. This modifier as placed makes it ambiguous. Is it only this parameter you can set at Postgres server start? Or is the only time you can set it at server start? If the latter (which it appears to be based on the original text), my edit is correct.

Reply:
```suggestion
Needs to be set high enough to cope with the maximum number of concurrent prepared transactions across the cluster due to explicit two-phase commits, CAMO, or Eager transactions. Exceeding the limit prevents a node from running a local two-phase commit or CAMO transaction and prevents all Eager transactions on the cluster. This parameter can only be set at Postgres server start.
```